staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,18 @@
+nav{
+    width: 20%;
+    float: left;
+    height: 100%;
+    display:inline-block;
+    }
+    
+    section{
+        width: 60%;
+        display:inline-block;
+        height: 100%;
+    } 
+    aside{
+        width: 20%;
```

Comment:
TIP group common rules in one single statement of grouped selectors.

SEE "Grouping Selectors" https://www.w3schools.com/css/css_syntax.asp

Reply:
Done! Thanks!